docs/starters.yml

```
@@ -4893,6 +4893,14 @@
     - Basic SEO, site metadata
     - Prettier
     - Social media links
+- url: https://gatsby-starter-default-nostyles.netlify.com/
+  repo: https://github.com/JuanJavier1979/gatsby-starter-default-nostyles
+  description: The default Gatsby starter with no styles.
+  tags:
+    - Styling:None
+   features:
```

```suggestion
 features:
```

Hey, looks like there is some bad indentation here. Could you fix this please?